server/src/main/java/org/apache/iotdb/db/qp/physical/crud/InsertRowsOfOneDevicePlan.java

```diff
@@ -29,26 +29,52 @@
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import java.util.ArrayList;
-import java.util.Collections;
+import java.util.Arrays;
+import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
 public class InsertRowsOfOneDevicePlan extends InsertPlan implements BatchPlan {
 
+  /**
+   * This class has some duplicated codes with InsertRowsPlan, they should be refined in the future
+   */
   boolean[] isExecuted;
+
   private InsertRowPlan[] rowPlans;
 
+  /**
+   * Suppose there is an InsertRowsOfOneDevicePlan, which contains 5 InsertRowPlans,
+   * rowPlans={InsertRowPlan_0, InsertRowPlan_1, InsertRowPlan_2, InsertRowPlan_3, InsertRowPlan_4},
+   * then the rowPlanIndexList={0, 1, 2, 3, 4} respectively. But when the InsertRowsOfOneDevicePlan
+   * is split into two InsertRowsOfOneDevicePlans according to different storage group in cluster
```

Comment:
All `InsertRowPlan` in `InsertRowsOfOneDevicePlan` belong to the same device, so they must belong to the same storage group, but they may be distributed into different data groups due to different time partition. You need to rephrase the comments here

Reply:
Thanks, I have updated this comment in the newest commit.